nodes/matrix/change_pivot.py

```diff
@@ -0,0 +1,63 @@
+import bpy
+from bpy.props import *
+from ... tree_info import keepNodeState
+from ... events import executionCodeChanged
+from ... base_types.node import AnimationNode
+
+inputTypeItems = [
```

Comment:
would be nice to have all 5 values here for each item. Most importantly the last number which will be used as identifier by Blender. In my experience this is more 'stable' when we extend nodes later.